openstack/castellum/templates/api-deployment.yaml

```diff
@@ -26,6 +26,26 @@ spec:
         checksum/configmap: {{ include "castellum/templates/configmap.yaml" $ | sha256sum }}
         checksum/secret: {{ include "castellum/templates/secret.yaml" $ | sha256sum }}
     spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                - key: release_name
+                  operator: In
+                  values:
+                  - castellum
+                - key: application
+                  operator: In
+                  values:
+                  - neutron
+                - key: component
+                  operator: In
+                  values:
+                  - api
```

Comment:
Besides the copy-paste error, this does not match the label structure that we have in this deployment. This labelSelector should probably match the one used by the Deployment (see `.spec.selector.matchLabels` in this same file).

Analogously for the other deployments.